packages/bot/src/typings.ts

```diff
@@ -130,7 +130,7 @@ export interface DiscordInteractionDataResolved {
   /** The Ids and Role objects */
   roles?: Record<string, DiscordRole>
   /** The Ids and partial Channel objects */
-  channels?: Record<string, Pick<DiscordChannel, 'id' | 'name' | 'type' | 'permissions'>>
+  channels?: Record<string, Pick<DiscordChannel, 'id' | 'name' | 'type' | 'permissions' | 'thread_metadata' | 'parent_id'>>
```

Comment:
Leaving this for future reference:
`** Partial Channel objects only have id, name, type and permissions fields. Threads will also have thread_metadata and parent_id fields.`
https://discord.com/developers/docs/interactions/receiving-and-responding#interaction-object-resolved-data-structure